### DISCUSSION TITLE
HEX COLOR

### DISCUSSION THREAD

**Discussion by Erissos** [TownyAdvanced/Towny#6686] [Ideas]
your plugin is also HEX colored like "#RRGGBB" or "&#RRGGBB", that's nice and finally, but some plugins are also #{RRGGBB} and that's a problem. For example, I set the city tag &7[#f334fd%s&7] from config.yml. When I reflect this on the scoreboard, it looks colorless like this: CITY: &7[#f334fd%s&7]

(I'm sorry my english is not good. all I want is to use HEX colors in #{RRGGBB} format.)

**Comment by LlmDl** [comment]
We use MiniMessage formatting for hex and what you see available is what you get to use.
